source/pages/practices/best_practices.rst

```diff
@@ -33,51 +22,47 @@ If you think your file format should be accepted by the repository, but is curre
 .. _file-size:
 
 File sizes and count
-_________________
+--------------------
 
 - **Maximum file size**: 4 GB using the online deposit form
 
-The maximum deposit data volume is 10 GB and the maximum number of files that can be uploaded via the onlone deposite is 10 files. If you want to deposit larger volumes of data or larger number of files in a single deposit, please contact us.
+The maximum deposit data volume is 10 GB and the maximum number of files that can be uploaded via the online deposit form is 10 files. If you want to deposit larger volumes of data or a larger number of files in a single deposit, please contact us.
```

Comment:
Does this maximum also apply when uploading via the REST API or only via the form?